mopro-ffi/tests/test_generated_bindings.rs

```diff
@@ -1,3 +1,21 @@
+#[cfg(feature = "gpu-benchmarks")]
+uniffi::build_foreign_language_testcases!(
+    // "tests/bindings/test_mopro_gen_benchmarks_report.swift", // Uncomment to generate benchmarks report, takes nearly 10 minutes
+    "tests/bindings/test_mopro_gpu_benchmarks.swift",
+    "tests/bindings/test_mopro.swift",
+    "tests/bindings/test_mopro.kts",
+    //    "tests/bindings/test_mopro.rb",
+    //    "tests/bindings/test_mopro.py",
+    "tests/bindings/test_mopro_keccak.swift",
+    // "tests/bindings/test_mopro_keccak.kts", // FIXME: java.lang.OutOfMemoryError: Java heap space
+    "tests/bindings/test_mopro_keccak2.swift",
```

Comment:
What are all these other stuff? Are you using more than `    "tests/bindings/test_mopro_gpu_benchmarks.swift",`?

Reply:
I think we can keep these two test under `#[cfg(feature = "gpu-benchmarks")]` and other swift tests under `#[cfg(not(feature = "gpu-benchmarks"))]`. What do you think? @FoodChain1028 

```
"tests/bindings/test_mopro_gen_benchmarks_report.swift",
"tests/bindings/test_mopro_gpu_benchmarks.swift",
```

Reply:
I'll make it like this.